Assets/DialogueBox.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DialogueBox : MonoBehaviour
+{
+    [SerializeField] GameObject dialogueBox;
+    public void showDialogueBox() {
+        dialogueBox.SetActive(true);
+        }
+    public void hideDialogueBox() {
+        dialogueBox.SetActive(false);
+        }
+}
```

Comment:
Unused, but if it's supposed to be used for the game object it's attached to, you should use this.SetActive() instead of passing itself trough a field.